src/components/Keypad/Keypad.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Keypad from './Keypad';
+
+describe('Keypad', () => {
+    let wrapper;
```

Comment:
```suggestion
    let subject;
```
Just a convention update.